src/main/java/uk/gov/gchq/magmacore/MagmaCore.java

```diff
@@ -24,11 +27,61 @@ public final class MagmaCore {
     private MagmaCore() {}
 
     /**
-     * Executes FusekiService or selected database example.
+     * Executes the selected database example.
      *
      * @param args Application arguments.
      */
     public static void main(final String[] args) {
```

Comment:
This is really part of the demo and is meant to allow people to run the different demos from one `main` - it should be moved as part of the switch to modules and either kept with the `examples` module or have its own module.

Reply:
We'll keep it as is until Magma Core is modularised. I'd prefer to have each of the demo classes executable directly, rather than relying on a single 'main' entry point with arguments.